bridges/zombienet/environments/rococo-westend/spawn.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
+
+trap "trap - SIGTERM && kill -9 -$$" SIGINT SIGTERM EXIT
+
+source "${BASH_SOURCE%/*}/../../utils/common.sh"
+source "${BASH_SOURCE%/*}/../../utils/zombienet.sh"
+
+logs_dir=$TEST_DIR/logs
+helper_script="${BASH_SOURCE%/*}/helper.sh"
+
+rococo_def=$POLKADOT_SDK_PATH/cumulus/zombienet/bridge-hubs/bridge_hub_rococo_local_network.toml
+start_zombienet $TEST_DIR $rococo_def rococo_dir rococo_pid
+echo
+
+rococo_init_log=$logs_dir/rococo-init.log
+echo -e "Setting up the rococo side of the bridge. Logs available at: $rococo_init_log\n"
+$helper_script init-asset-hub-rococo-local >> $rococo_init_log 2>&1
```

Comment:
Makes sense to have this part modular somehow since some tests need it and some don't. But I would keep it as part of the environment. Added a flag `--init` in order to control this.